mlrun/api/main.py

```diff
@@ -32,6 +34,20 @@
 app.include_router(api_router, prefix="/api")
 
 
+@app.exception_handler(mlrun.errors.HTTPStatusableError)
+async def http_statusable_error_handler(
```

Comment:
```suggestion
async def http_status_error_handler(
```
Also suggest:
`HTTPStatusableError` -> `HTTPStatusError`